src/vizier/services/adaptive_export/internal/pxl/queryfor_test.go

```
@@ -0,0 +1,217 @@
+	"errors"
+	"strings"
+	"testing"
+	"time"
+
+	"px.dev/pixie/src/vizier/services/adaptive_export/internal/anomaly"
+)
+
+// fixed reference time for deterministic relStart computation.
+var (
+	fixedNow   = time.Date(2026, 5, 9, 15, 23, 44, 0, time.UTC)
+	fixedStart = fixedNow.Add(-5 * time.Minute)             // ATTACK − 5 min
```

golangci-lint (2.12.1)

[error] File is not properly formatted

(gci)